lte/gateway/docker/README.md

```diff
@@ -61,14 +61,12 @@ and dirty way to run the containers locally.
 
 ### Running the S1AP integration tests against the containerized AGW
 
-At present, of the S1AP integration tests, it has been verified that `test_attach_detach` can be run versus the containerized AGW.
-To run this test, development mode needs to be enabled when starting the containers, which can be done by using an override
-file `docker-compose.dev.yaml`:
+At present, of the S1AP integration tests, it has been verified that most of the tests can be run versus the containerized AGW;
```

Comment:
Can we be more precise here and maybe just say which tests don't work? If not, nitpicky suggestion for a shorter sentence:
> It has been verified that most of the S1AP tests can be run against the containerized AGW.

Reply:
I agree with @jheidbrink on both counts. If you don't want to add a list of tests, I prefer the suggested sentence

Reply:
I do not think we want a constantly changing list of tests in the readme file.  I can delete the comment completly if you prefer.

Reply:
Yes, deleting it sounds good actually. Maybe replacing it with a sentence like "To run the tests, first start the docker containers:" would be good.

Reply:
If modifying this, we should also change the following below. "The test VM can then be set up and the test executed by following" -> "The test VM can then be set up and the tests executed by following"